analaysis/plant/plant_functional_types/plant_functional_type_species_classification_base.R

```diff
@@ -0,0 +1,182 @@
+#' title: Plant functional type species classification base
+#'
+#' description: |
+#'     This script classifies species into plant functional types (PFTs).
+#'     At the moment, these PFTs only contain trees.
+#'     This classification is predominantly based on the one provided by
+#'     Kohler and Huth (1998; DOI https://doi.org/10.1016/S0304-3800(98)00066-0)
+#'     and contains the following PFTs:
+#'     -emergent trees
+#'     -overstory trees
+#'     -understory trees
+#'     -pioneer trees
+#'     For the understory PFT, the species list is expanded by also including
+#'     the species listed in group 9 (small shade trees) in
+#'     Phillips et al. (2000; DOI https://doi.org/10.1016/S0378-1127(00)00666-6).
+#'     This combined PFT species classification is then applied to the
+#'     SAFE census dataset.
+#'     The output of this script generates a CSV file containing a list of
+#'     species and their respective PFT.
+#'     This CSV file can then be loaded when working with other datasets
+#'     (particularly those related to updating T model parameters).
+#'     In a follow up script, the remaining species that have not been assigned
+#'     a PFT yet will be assigned into one based on their species maximum height
+#'     relative to the PFT maximum height.
+#'
+#' VE_module: Plant
+#'
+#' author:
+#'   - name: Arne Scheire
+#'
+#' status: final
+#'
+#'
+#' input_files:
+#'   - name: tree_census_11_20
+#'     path: ve_data_science\data\primary\plant\tree_census
+#'     description: |
+#'       https://doi.org/10.5281/zenodo.14882506
+#'       Tree census data from the SAFE Project 2011–2020.
+#'       Data includes measurements of DBH and estimates of tree height for all
+#'       stems, fruiting and flowering estimates,
+#'       estimates of epiphyte and liana cover, and taxonomic IDs.
+#'
+#' output_files:
+#'   - name: plant_functional_type_species_classification_base
+#'     path: ve_data_science\data\derived\plant\plant_functional_type
+#'     description: |
+#'       This CSV file contains a list of species and their respective PFT.
+#'       This CSV file can be loaded when working with other datasets
+#'       (particularly those related to updating T model parameters).
+#'       In a follow up script, the remaining species that have not been assigned
+#'       a PFT yet will be assigned into one based on
+#'       their species maximum height relative to the PFT maximum height.
+#'
+#' package_dependencies:
+#'     - readxl
+#'
+#' usage_notes: |
+#'   If PFT species classification is updated in the future,
+#'   this script should be the starting point.
+#'   File directories still need to be converted to relative paths.
+#' ---
+
+
+# Load packages
+
+library(readxl)
+
+# Load SAFE tree census data
+
+tree_census_11_20 <- read_excel("C:\\Users\\schei\\VE_Data_Science\\
+                              ve_data_science\\data\\primary\\plant\\
+                              tree_census\\tree_census_11_20.xlsx",
```

Comment:
This should be a relative path: "../../../data/primary/plant/tree_census/tree_census_11_20.xlsx`

Can you try using those forward slashes in your script? I don't recall if R is smart enough to handle OS differences in path structures?